repo2docker/buildpacks/julia.py

```diff
@@ -69,4 +107,10 @@ def get_assemble_scripts(self):
         )]
 
     def detect(self):
+        """
+        Check if current repo should be built with the Julia Build pack
+
+        super().detect() is called if Julia Build pack should be used.
```

Comment:
I think similar to the R build pack, we should actually *not* call super().detect() here, for the same reason as on the R build pack (we want to use Julia even if there is no environment.yml file). Whenever I'm doing doc fixes I'm also happy to fix bugs so documenting them is easier, but happy to deal with that in a separate PR too!